crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -546,7 +548,7 @@ pub fn view_3d(
                 if space_view_id == &query.space_view_id {
                     state
                         .state_3d
-                        .reset_camera(&state.bounding_boxes.accumulated, scene_view_coordinates);
+                        .reset_camera(&state.bounding_boxes.current, scene_view_coordinates);
```

Comment:
What about the other place `reset_camera` is called from?

Maybe it's better to pass in the full `SceneBoundingBoxes` to `reset_camera`?

Reply:
good point actually